src/App.vue

```diff
@@ -1,26 +1,30 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
-  <HelloWorld msg="Welcome to Your Vue.js App"/>
+  <div id="nav">
```

Comment:
The Navigation should become a component in itself. And the styling that you have added should be added inside that.

Reply:
Hi! I have not created any component right now. That is why it is here.
Once I add a component library and tailwind support I will create separate component for Nav

Reply:
Ok, no problem.